deploy/minibroker.sh

```diff
@@ -30,7 +30,7 @@ if ! helm version 1> /dev/null 2> /dev/null; then
   exit 1
 fi
 
-git_root="$(git rev-parse --show-toplevel)"
+script_dir="$(dirname "$(realpath "${BASH_SOURCE}")")"
```

Comment:
I think you technically want `${BASH_SOURCE[0]}` but `${BASH_SOURCE}` implicitly means the first one?
(I don't think `realpath` is going to be on macOS, but we probably don't care.)

Reply:
I slightly care... but macOS users can install `coreutils` using Homebrew.